tests/framework/checks/checks.go

```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 	"time"
 
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
```

Comment:
nit: move this in the proper section below

Reply:
done!